db/seeds.rb

```diff
@@ -10,32 +10,41 @@
 
 RootNode.create state: 'published'
 
+news = Section.find_or_create_by!(name: "news")
+
 topic = Topic.find_or_create_by!(name: "Business")
 topic.summary = 'The business section covers a range of business-related topics.'
 topic.save
 
+news1 = NewsArticle.find_or_create_by!({
+                                           name: "Business News",
+                                           section: topic,
+                                           parent: news.home_node,
+                                           state: :published,
+                                       })
+news1.revise!(content_body: 'foobar')
```

Comment:
If you are planning to run seeds on prod then this should idempotent so maybe only revise if news1 was just created, i.e. 
````
NewsArticle.find_or_create_by!({ ...etc ....}).do |news_article| 
  news_article.revise!(content_body: 'foobar') 
end
````
(FYI - a block passed to `find_or_create_by` is only run on create.)